apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/condition/repository/ConditionCodeRepository.java

```diff
@@ -12,4 +14,9 @@ public interface ConditionCodeRepository extends JpaRepository<ConditionCode, St
     long checkId(@Param("id")String id);
     @Query("SELECT count(*) FROM ConditionCode c WHERE c.conditionShortNm=:name")
     long checkConditionName(@Param("name") String name);
+
+    @Query("SELECT c FROM ConditionCode c WHERE c.id LIKE %:id% OR c.conditionShortNm LIKE %:conditionShortNm% OR c.progAreaCd LIKE %:progAreaCd% OR c.familyCd LIKE %:familyCd% OR c.coinfectionGrpCd LIKE %:coinfectionGrpCd% OR c.nndInd =:nndInd OR c.investigationFormCd LIKE %:investigationFormCd% OR c.statusCd =:statusCd")
+    Page<ConditionCode> findByField(@Param("id")String id, @Param("conditionShortNm") String conditionShortNm, @Param("progAreaCd") String progAreaCd,
+                                    @Param("familyCd")String familyCd, @Param("coinfectionGrpCd")String coinfectionGrpCd, @Param("nndInd")Character nndInd, @Param("investigationFormCd") String investigationFormCd,
+                                    @Param("statusCd")Character statusCd, Pageable pageable);
 }
```

Comment:
If the UI is going to be a single search box, this would be changed to have a single parameter "searchText" or "searchQuery" and use that value in each of the `LIKE`s